front/app/components/admin/Graphs/_components/Container.tsx

```diff
@@ -29,6 +29,14 @@ interface Props {
   children: React.ReactElement;
 }
 
```

Comment:
Dirty hack to disable annoying `recharts` warnings. I tried everything else but it would either break functionality or unit tests

Reply:
You mean the responsive container? Yeah but then we have to do that check in every graph component AND inside of the container component. Might not be that much work but seems like a stupid design even though this works fine

Reply:
yeah, i was thinking on doing it only on the this file, but don't worry about it, its fine

Reply:
That would be cool but unfortunately it's not possible, you would have to pass the numbers directly to the `BarChart` or whatever graph component you nest inside of this `Container`

Reply:
but wouldn't be just checking if `parsedWidth` or `parsedHeight` contain a % sign?